modernomad/settings/common.py

```diff
@@ -94,8 +94,14 @@
     'django.contrib.auth.backends.ModelBackend'
 )
 
-EMAIL_BACKEND = 'modernomad.backends.MailgunBackend'
 MAILGUN_API_KEY = env('MAILGUN_API_KEY', default='')
+if MAILGUN_API_KEY:
+    EMAIL_BACKEND = 'modernomad.backends.MailgunBackend'
+    # This should only ever be true in the production environment. Defaults to False.
+    MAILGUN_CAUTION_SEND_REAL_MAIL = env('MAILGUN_CAUTION_SEND_REAL_MAIL', default='')
```

Comment:
I'm not too sure about this approach, though if you feel like this solves an immediate problem and would like this merged I'm pragmatic about it. 

I don't fully understand the test mode thing that you mentioned earlier. The way I understood mailgun was that you have your production domain, and then you also have your sandbox. The sandbox api key would be used for staging and the production api key would be used for production. 

Is it that you are afraid there might be a mistake that these keys are being mixed up when making the environment configuration for production and staging respectively?

Reply:
Found it! https://documentation.mailgun.com/en/latest/user_manual.html#sending-in-test-mode

Rather than relying on testmode it seems better to use the sandbox in my opinion.

Reply:
Resolved below I believe!